.devops/main.Dockerfile

```diff
@@ -9,14 +9,14 @@ WORKDIR /app
 
```

Comment:
The name of this Dockerfile also need to be changed

Reply:
Done, thanks